content/working_with/workflows/built-in-workflows.md

```diff
@@ -217,86 +217,96 @@ This workflow simply calls the `stop` workflow, followed by `start`.
 
 **Workflow name:** **`heal`**
 
-**Workflow description:** Reinstalls the whole subgraph of the system topology by applying the `uninstall` and `install` workflows' logic respectively. The subgraph consists of all the node instances that are contained in the compute node instance which contains the failing node instance and/or the compute node instance itself. Additionally, this workflow handles unlinking and establishing all affected relationships in an appropriate order.
+**Workflow description:** Heals node-instances of a deployment, or a subset of them. Run the `check_status` and `heal` interfaces for each selected instance, and in case those operations fail or are not declared, reinstall the node instance. Instances that pass their `check_status` call are not healed or reinstalled.
```

Comment:
'Run' should probably be 'Runs' (as 'Heals' is there rather than 'Heal')

Reply:
Same for 'reinstall' (should be 'reinstalls' to fit with 'Heals')

Reply:
I prefer the `Heal` wording, then